Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM phusion/baseimage:0.9.18 
+FROM phusion/passenger-ruby19:0.9.18
```

Comment:
Am I right in thinking this Dockerfile represents the `railsweb` container? Just making sure.